webui/react/src/pages/F_ExpList/glide-table/GlideTable.tsx

```diff
@@ -259,18 +259,23 @@ export const GlideTable: React.FC<GlideTableProps> = ({
     (cell: Item): GridCell => {
       const [col, row] = cell;
 
+      const customCell: GridCell = {
+        allowOverlay: true,
+        copyData: '',
+        data: { kind: 'spinner-cell' },
+        kind: GridCellKind.Custom,
+      };
+
+      if (row >= data.length) {
+        return customCell;
+      }
```

Comment:
question: should this just check to see if it's undefined as opposed to whether row is within the data length boundary? AKA:
```
if (!data[row]) return customCell;
```

Reply:
i first wrote that way and change it to the current one. I just thought `row >= data.length` is more meaningful to code readers. I'll switch to `!data[row]` with a comment

Reply:
I agree that it's more meaningful to readers. Better to be explicit here.

Reply:
added comment, so reader will know the context of the early return guard